### PR TITLE
Added html for section 1

### PR DESCRIPTION
**Pull request by Alpha787**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+  <title>Document</title>
+</head>
+
+<body>
+  <header>Header</header>
+  <section class="section" id="warranty">
+    <div class="wrapper">
+      <div class="text-block">
+        <h1 class="title">Peaceful shopping mindful money</h1>
+        <p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
```
Comment (KoloskovaAlla) [inline index.html:16]: Replace with
````
<p class="copy">
     See how we can help with making your shopping experience and money management more ZEN.
</p>

File: index.html [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+        <div class="links">
+          <a class="link" href="https://apple.com">
+            <img src="./assets/icons/apple.svg" alt="Apple icon">
+          </a>
+          <a class="link" href="https:/google.com">
+            <img src="./assets/icons/google.svg" alt="Google icon" class="">
```
Comment (KoloskovaAlla) [inline index.html:22]: Recommend to replace it with
````
<img src="./assets/icons/google.svg" alt="Google icon" class="" />

File: index.html [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
+        </div>
+      </div>
+      <div class="image-block">
+        <img src="./assets/images/1.jpg" alt="Bag and device" class="">
```
Comment (KoloskovaAlla) [inline index.html:26]: The same
````
<img />

**Review by KoloskovaAlla — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -1,18 +1,39 @@
...
+          </a>
+          <a class="link" href="https:/google.com">
+            <img src="./assets/icons/google.svg" alt="Google icon" class="" />
+        </div>
+      </div>
+      <div class="image-block">
+        <img src="./assets/images/1.jpg" alt="Bag and device" class="" />
+      </div>
+    </div>
+  </section>
```
Comment (nagorapro) [inline index.html:31]: Use BEM methodology
Use "download" class name
Use "download" id name

**Pull request merged**